app/src/main/java/samyups/example/trainingss/data/Dog.kt

```diff
@@ -1,7 +1,11 @@
 package samyups.example.trainingss.data
 
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "Dog Breeds")
 data class Dog(
-    val breed: String,
+    @PrimaryKey val breed: String,
```

Comment:
PrimaryKey is like an ID for your entry in the database. For example if the government has a database of all Americans, your PrimaryKey will probably be your Social Security number, it wouldn't be your ethnicity. There's lots of Dominicans in the US, and lots of Pakistani's in the US, just like there could eventually be lots of Husky's or Corgi's in your database. You need a unique identifier as your Primary Key. I would recommend you look up how to generate a Primary Key in Room. It doesn't matter what the key is, in fact it will probably be randomly generated. What this means though is that you will have to add an ID field to your `Dog` data class that will be the primary key, and you will have to ask the database to generate the ID when you go to enter it, and make sure you set that ID to the ID in your dataclass. Now your Dog object in your List of Dogs is associated with an entry in your database. 

P.S. Look up the term POJO. It stands for Plain Old Java Object, and it is used to refer to data that is represented in Java. In this case, your POJO is a Dog object. The ID will associate your POJO with an entry in the Database (which is not a POJO). You need some way to connect your POJO's to your database.